lib/workers/repository/cache.ts

```diff
@@ -75,7 +75,8 @@ async function generateBranchCache(branch: BranchConfig): Promise<BranchCache> {
       isModified,
       upgrades,
     };
-  } catch (err) {
+  } catch (error) {
+    const err = error.err || error; // external host error nests err
```

Comment:
Shouldn't we do a `externalHostError` type check?

Reply:
There might one day be other types we nest `err` :)

Reply:
Then we should create a base class to check for 😉 .

We should think of a better solution anyways, because the `any`  error type in catch blocks caused so many bugs, because of missing compile time checks.

Reply:
I want to merge this right now though to prevent errors in the app